### ISSUE TITLE
sc_0221: no gate animation during Lasse's cutscene

### ISSUE THREAD

**Issue by AcipenserSturio** [Helco/zzio#279]
Reproducing should be as easy as opening sc_0221 and walking up to the gate. Lasse passes right through

Amy interaction with the gate works as intended, however

**Comment by Helco** [comment]
That is correct, Biped NPCs should generally open doors, but this is not implemented yet.
